runtime/gc_modron_standard/ContinuationObjectBufferStandard.cpp

```diff
@@ -102,3 +105,37 @@ MM_ContinuationObjectBufferStandard::reinitializeForRestore(MM_EnvironmentBase *
 	return true;
 }
 #endif /* defined(J9VM_OPT_CRIU_SUPPORT) */
+
+void
+MM_ContinuationObjectBufferStandard::iterateAllContinuationObjects(MM_EnvironmentBase *env)
+{
+	MM_GCExtensions *extensions = MM_GCExtensions::getExtensions(env);
+	MM_HeapRegionDescriptorStandard *region = NULL;
+	GC_HeapRegionIteratorStandard regionIterator(extensions->heapRegionManager);
+
+	/* to make sure that previous pruning phase of continuation lis(scanContinuationObjects())t is complete */
+	env->_currentTask->synchronizeGCThreads(env, UNIQUE_ID);
+
+	while (NULL != (region = regionIterator.nextRegion())) {
+		MM_HeapRegionDescriptorStandardExtension *regionExtension = MM_ConfigurationDelegate::getHeapRegionDescriptorStandardExtension(env, region);
+		for (uintptr_t i = 0; i < regionExtension->_maxListIndex; i++) {
+			MM_ContinuationObjectList *list = &regionExtension->_continuationObjectLists[i];
+			if (NULL != list->getHeadOfList()) {
+				if (J9MODRON_HANDLE_NEXT_WORK_UNIT(env)) {
+
+					omrobjectptr_t object = list->getHeadOfList();
+					while (NULL != object) {
+						omrobjectptr_t next = extensions->accessBarrier->getContinuationLink(object);
+#if JAVA_SPEC_VERSION >= 19
```

Comment:
we should probably move  #ifdef to an outer spot(s)...
around the whole method and even around the new code block in RootScaner

Reply:
move  #ifdef  into RootScaner